Sources/Views/MusicProgressSliderCell.m

```diff
@@ -0,0 +1,55 @@
+//
+//  MusicProgressSlider.m
+//  Hermes
+//
+//  Created by Xinhong LIU on 19/4/15.
+//
+//
+
+#import "MusicProgressSliderCell.h"
+
+@implementation MusicProgressSliderCell
+
+@synthesize knobImage = _knobImage;
+
+- (NSImage *)knobImage {
+  return _knobImage;
+}
+
+- (void)setKnobImage:(NSImage *)knobImage {
+  _knobImage = knobImage;
+  if (_knobImage == nil) {
+    // default knob image
+    _knobImage = [NSImage imageNamed:@"knob.png"];
```

Comment:
You should be able to use the public setter: `self.knobImage = [NSImage imageNamed:@"knob.png"]`